packages/mdc-slider/_mixins.scss

```diff
@@ -20,24 +20,52 @@
 @import "./variables";
 
 @mixin mdc-slider-track-color($color) {
-  .mdc-slider__track {
-    @include mdc-theme-prop(background-color, rgba(mdc-theme-prop-value($color), $mdc-slider-track-opacity));
+  &:not(.mdc-slider--disabled) {
+    .mdc-slider__track {
+      @include mdc-theme-prop(background-color, rgba(mdc-theme-prop-value($color), $mdc-slider-track-opacity));
+    }
   }
 }
 
 @mixin mdc-slider-track-fill-color($color) {
-  .mdc-slider__track-fill {
-    @include mdc-theme-prop(background-color, $color);
+  &:not(.mdc-slider--disabled) {
+    .mdc-slider__track-fill {
+      @include mdc-theme-prop(background-color, $color);
+    }
   }
 }
 
 @mixin mdc-slider-thumb-color($color) {
-  .mdc-slider__thumb {
-    @include mdc-states($color);
+  &:not(.mdc-slider--disabled) {
+    .mdc-slider__thumb {
+      @include mdc-states($color);
+    }
+
+    .mdc-slider__thumb-handle {
+      @include mdc-theme-prop(fill, $color);
+    }
+  }
+}
+
+@mixin mdc-slider-disabled($color) {
```

Comment:
I'd rename this to more accurately describe that it modifies the color. Something like `mdc-slider-disabled-color` or similar.

Reply:
Done!